UI/Web/src/app/settings/_components/setting-multi-check-box/setting-multi-check-box.component.ts

```diff
@@ -0,0 +1,159 @@
+import {
```

Comment:
We should probably move this to common folder or something since it's more of a library component.

Reply:
Okay, we can leave here for now and if we start using elsewhere, move them.